README.md

```diff
@@ -1,71 +1,102 @@
-# pspdev
+# PSPDEV
 
 [![GitHub Workflow Status](https://img.shields.io/github/workflow/status/pspdev/pspdev/CI?label=CI&logo=github&style=for-the-badge)](https://github.com/pspdev/pspdev/actions?query=workflow%3ACI)[![GitHub Workflow Status](https://img.shields.io/github/workflow/status/pspdev/pspdev/CI-Docker?label=CI-Docker&logo=github&style=for-the-badge)](https://github.com/pspdev/pspdev/actions?query=workflow%3ACI-Docker)[![Docker Pulls](https://img.shields.io/docker/pulls/pspdev/pspdev?style=for-the-badge)](https://hub.docker.com/r/pspdev/pspdev/tags)
 
 Main PSP Repo for building the whole `PSP Development` environment in your local machine.
 
-This program will automatically build and install the whole compiler and other tools used in the creation of homebrew software for the Sony PlayStation Portable® videogame system.
+## Table of Contents
+
+- [PSPDEV](#pspdev)
+  - [Table of Contents](#table-of-contents)
+  - [Up and running](#up-and-running)
+  - [What these scripts do](#what-these-scripts-do)
+  - [Requirements](#requirements)
+    - [Ubuntu/Debian](#ubuntudebian)
+    - [Fedora](#fedora)
+    - [Arch](#arch)
+    - [OSX](#osx)
+  - [Docker generation](#docker-generation)
+  - [Extra steps](#extra-steps)
+    - [macOS](#macos)
+    - [Local package builds](#local-package-builds)
+  - [Thanks](#thanks)
+
+This program will automatically build and install the whole compiler and other tools used in the creation of Homebrew software for the Sony PlayStation Portable® video game system.
+
+## Up and running
+
+You can get started very quickly by grabbing the latest development pre-releases from the [releases' page for your platform](https://github.com/pspdev/pspdev/releases) and extract them to the `pspdev` directory in your `path`.
```

Comment:
I would say home directory instead of path here.

The use would then need to add the following to their `~/.bashrc` file:
```bash
export PSPDEV=~/pspdev
export PATH=$PATH:$PSPDEV/bin
```

Reply:
Ah, I did think about that, but it wouldn't be applicable to a person who wants to use the windows build. 
Should I add this anyways?

Reply:
It seems to me like this page strongly suggests using `/usr/local/pspdev` as your installation directory. Should we say the home directory instead?

Reply:
Yeah, I would like to see that changed.

It should be clear from this part what the user should do and that's currently missing. Platform specific instructions are acceptable. I think Mac users will need to set it in `~/.bash_profile` or in their zsh profile, whatever that is called.